cpu/stm32/stmclk/stmclk_f2f4f7.c

```diff
@@ -38,8 +38,6 @@
  * @name    PLL configuration
```

Comment:
unrelated

Reply:
I think it is: this file is adapted with the removal of code related to f1 and f3. And the name is changed to reflect that now it only contains code related to f2f4f7. Keeping fx suffix makes no sense since it doesn't apply anymore to all Fx families.

Reply:
Ah ok